### PR TITLE
Make manual easier

### PR DESCRIPTION
**Pull request by kfu02**
## Description
Making it easier to use the manual control node and service. 
* Adds script to avoid launching rqt every time we want to test manual.
* Deletes manual.launch.py and replaces it with a flag in soccer.launch.py. 
* Adds comments and slight changes to makefile and soccer.launch.py.

## Associated / Resolved Issue
Resolves # or [ClickUp card](https://app.clickup.com/t/34bj9jn)

### Test Case 1
1. Follow instructions in this PR's `doc/RunningOnFieldComp.md`
2. Make sure I didn't break any of the other `make run-sim` derivatives.

**Expected result:** Docs are clear and describe what's happening. `make run-sim, make run-sim-ex, etc.` still work as intended.

### CODE REVIEW

File: launch/soccer.launch.py [hunk 1]
```diff
@@ -38,12 +35,15 @@ def generate_launch_description():
     bringup_dir = Path(get_package_share_directory("rj_robocup"))
     launch_dir = bringup_dir / "launch"
 
+    # TODO(Alex): why are there two defaults? here and at bottom of file in
```
Comment (kasohrab) [inline launch/soccer.launch.py]: You can delete this. Apparently you need both, one is for local values (launch configuration), the other exposes it to the user to change at launch time: https://docs.ros.org/en/galactic/Tutorials/Intermediate/Launch/Using-Substitutions.html#substitutions-example-launch-file. 

This answer explains it better than ros2 documentation:
https://answers.ros.org/question/322874/ros2-what-is-different-between-declarelaunchargument-and-launchconfiguration/

So maybe we only need the declarelaunchargument, but I'm pretty sure we need both.

Comment (kasohrab) [reply launch/soccer.launch.py]: Seems like a dumb ROS quirk to me lol

File: util/manual_control_connect.bash [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+do
+    case "${flag}" in
+        h) display_help;;
+        r) robot_id=${OPTARG};;
+        c) controller_uuid=${OPTARG};;
+        C) connect=${OPTARG};;
+    esac
+done
+
+cmd="ros2 service call /select_manual rj_msgs/srv/SetManual \"{robot_id: $robot_id, controller_uuid: $controller_uuid, connect: $connect}\""
```
Comment (kasohrab) [inline util/manual_control_connect.bash:39]: This is a big improvement from before, but I would add a card to have the ui call this script.

Comment (kasohrab) [reply util/manual_control_connect.bash:39]: (When you input a manual id and launch in manual mode)

**Review by kasohrab — CHANGES_REQUESTED** [review]
Looks great! Some comments before merging.

File: util/manual_control_connect.bash (hunk 2 above)
Comment (kfu02) [reply util/manual_control_connect.bash:39]: way ahead of you: https://app.clickup.com/t/2tdhtqv

**Review by kasohrab — APPROVED** [review]
Looks good to me.

**Comment by kfu02** [comment]
currently blocked by #1928 , I would like to make the launch process for sim vs real much better before merging these changes.

on second thought, #1928 will take a while and this change is nice. merging.

**Pull request merged**